docs/source/1.0/spec/core/lexical-structure.rst

```diff
@@ -283,18 +271,13 @@ Shape IDs are formally defined by the following ABNF:
 
 .. productionlist:: smithy
     shape_id               :`root_shape_id` [`shape_id_member`]
-    root_shape_id          :`absolute_shape_id` / `identifier`
-    absolute_shape_id      :`namespace` "#" `identifier`
+    root_shape_id          :`absolute_root_shape_id` / `identifier`
+    absolute_root_shape_id :`namespace` "#" `identifier`
     namespace              :`identifier` *("." `identifier`)
     identifier             :(ALPHA / "_") *(ALPHA / DIGIT / "_")
     shape_id_member        :"$" `identifier`
     LOALPHA                :%x61-7A ; a-z
```

Comment:
This isn't used anywhere

Reply:
`absolute_root_shape_id`? It's used to define use statements:

```
use_section   = *(use_statement)
use_statement = "use" ws absolute_root_shape_id br
```

Reply:
Oh, LOALPHA